modules/snooker/src/main/ts/ephox/snooker/api/TableFill.ts

```diff
@@ -55,6 +62,16 @@ const cloneFormats = function (oldCell: SugarElement, newCell: SugarElement, for
   }).getOr(newCell);
 };
 
+const copyAppropriateAttributes = <T extends HTMLElement> (original: SugarElement<T>, clone: SugarElement<T>): void => {
+  Arr.each(Obj.keys(transferableAttributes), (attributeName: string) => {
```

Comment:
This should also probably just be using `Obj.each` instead of `Arr.each` and `Obj.keys`.